workflows_core/api/api.py

```diff
@@ -405,29 +405,82 @@ def _trigger(
         ).json()
 
     @retry()
-    def _progress(
+    def _update_workflow_progress(
         self,
         workflow_id: str,
         worker_number: int = 0,
-        step=0,
+        step: int = 0,
         n_processed: int = 0,
         n_total: int = 0,
     ):
         """
         Tracks Workflow Progress
         """
-        params = dict(
-            worker_number=worker_number,
-            step=step,
-            n_processed=n_processed,
-            n_total=n_total,
-        )
-        # print the params to see what is happening here
-        logger.debug("adding progress...")
-        logger.debug(params)
         response = requests.post(
```

Comment:
Why are you removing my logs?

Reply:
ideally we dont add logging in the api, there should be a method that turns on logging for all requests or not, imo